lib/components/notifications-component.js

```diff
@@ -16,8 +16,13 @@ export default class NotificationsComponent extends BaseContainer {
 		return text;
 	}
 	selectCommentByText( commentText ) {
-		const commentSelector = by.xpath( `//div[normalize-space(text())='${commentText}']` );
-		return driverHelper.clickWhenClickable( this.driver, commentSelector );
+		const self = this;
+		const commentSelector = by.css( 'div > div.wpnc__text-summary > div:nth-child(1)' );
+		this.driver.findElement( by.css( 'div.wpnc__excerpt' ) ).getText().then( function( selectedCommentText ) {
+			if( selectedCommentText === commentText ) {
```

Comment:
what happens where the comment text doesn't match? It seems like this will result in different behaviour and may be confusing to work out what is going on...